### PR TITLE
Array unit tests

### PR DESCRIPTION
**Pull request by devsmt**
fixed and tested 2 simple edge cases in core array functions (sorry people, this is my first pull request!)

### CODE REVIEW

File: tests/arr.php [hunk 1]
```diff
@@ -492,6 +492,13 @@ public function test_sort_invalid_direction($data, $expected)
 		$this->assertEquals(Arr::sort($data, 'info.pet.type', 'downer'), $expected);
 	}
 
+	public function test_sort_empty()
+	{
+		$expected = array();
+		$output = Arr::Sort(array(), 'test', 'test');
```
Comment (frankdejonge) [inline tests/arr.php]: This should be lowercase 'sort'.

**Comment by devsmt** [comment]
resolved

File: tests/arr.php [hunk 2]
```diff
@@ -492,6 +492,17 @@ public function test_sort_invalid_direction($data, $expected)
...
 	}
 
+	public function test_sort_empty()
+	{
+		$expected = array();
+<<<<<<< HEAD
+		$output = Arr::sort(array(), 'test', 'test');
+=======
+		$output = Arr::Sort(array(), 'test', 'test');
+>>>>>>> 85988a20a4cdc8acfaa251be68aac253939f0aed
```
Comment (frankdejonge) [inline tests/arr.php]: oops? :P

Comment (devsmt) [inline tests/arr.php]: yes... any suggestion to clean it up?

Comment (jschreuder) [inline tests/arr.php]: It's a Git merge conflict, you need to decide which to use... (the lowercase one obviously)

File: classes/arr.php [hunk 3]
```diff
@@ -481,6 +481,11 @@ public static function sort($array, $key, $order = 'asc', $sort_flags = SORT_REG
 			throw new \InvalidArgumentException('Arr::sort() - $array must be an array.');
 		}
 
+		if( empty($array) )
```
Comment (frankdejonge) [inline classes/arr.php]: small style fix -> if (empty))
The rest seems to be correct

File: tests/format.php [hunk 4]
```diff
@@ -151,4 +151,10 @@ public function test__from_xml()
 
 		$this->assertEquals(Format::forge($expected)->to_php(), Format::forge($xml, 'xml')->to_php());
 	}
+
+	function test_to_array_empty(){
```
Comment (frankdejonge) [inline tests/format.php]: and the opening bracket needs to be on a new line

Comment (devsmt) [inline tests/format.php]: do you use an automated tool to enforce formatting? any help with formatting issues will benefit the project of more occasional contributions from casual users used to different conventions (like me...)

Comment (frankdejonge) [inline tests/format.php]: I don't maybe others do. But it's just following our style guide: http://docs.fuelphp.com/general/coding_standards.html
